.tool-versions

```diff
@@ -1,4 +1,4 @@
-nodejs 16.17.0
+nodejs 22.12.0
```

Comment:
Is there any dependency in the code bound to version 16?

Reply:
Dependencies aren't bound to a specific node versions but typically when you migrate your node version you have access to more features and the performance is also improved.